tools/python/liveanalytics_migration_scripts/targets/timestream_for_influxdb/live_migration_guide.md

```diff
@@ -0,0 +1,323 @@
+# Live Migration Guide
+
+This guide explains how to perform **live migrations** when moving your applications from **Amazon Timestream for LiveAnalytics** to [**Timestream for InfluxDB**](https://docs.aws.amazon.com/timestream/latest/developerguide/timestream-for-influxdb.html).
+
+The migration has three high‑level phases:
+
+1. **Data Migration** – continuously back‑fill and replicate historical + live records from Timestream for LiveAnalytics into Timestream for InfluxDB using batched [Line Protocol](https://docs.influxdata.com/influxdb3/cloud-dedicated/reference/syntax/line-protocol/) ingestion.
+2. **Application Migration** – refactor your application code, queries, dashboards and alerting rules so clients read from and write to Timestream for InfluxDB.
+3. **Clean Up** – de‑commission your legacy Timestream for LiveAnalytics resources once data and traffic have been fully cut over.
+
+---
+## Table of Contents
+
+0. [Prerequisites](#prerequisites)
+1. [Step 1 · Data Migration](#step-1--data-migration)
+   - [Quick start](#quick-start)
+   - [Tracking Migration Progress](#tracking-migration-progress)
+   - [Batch Directory Structure](#batch-directory-structure)
+2. [Step 2 · Application Migration](#step-2--application-migration)
+   - [Migration Strategy](#migration-strategy)
+   - [Coordination and Planning](#coordination-and-planning)
+   - [Implementation Guide](#implementation-guide)
+3. [Step 3 · Clean Up](#step-3--clean-up)
+   - [Checklist](#checklist)
+---
+
+## Prerequisites
+
+Before starting the migration, ensure you have the necessary tools and dependencies installed. Follow the installation steps at **[README.md#Installation](../../../README.md#installation)**.
+
+- Migrating to <b>InfluxDB V2</b>
+
+    Define the following environment variables:
+    ```
+    export INFLUXDB_V2_URL="https://influxdb_v2_url:8086"
+    export INFLUXDB_V2_ORG="org"
+    export INFLUXDB_V2_TOKEN="xxx"
+    ```
+
+- Migrating to <b>InfluxDB V3</b>
+
+    InfluxDB V3 supports ingestion through the [V2 write API](https://docs.influxdata.com/influxdb3/enterprise/write-data/compatibility-apis/).
+
+    Define the following environment variables, omitting `INFLUXDB_V2_ORG` (concept of organizations do not apply in V3):
+    ```
+    export INFLUXDB_V2_URL="https://influxdb_v3_url:8181"
+    export INFLUXDB_V2_TOKEN="xxx"
+    ```
+
+    > **Large datasets (≥1 PB)** – LP ingestion can be slow. The InfluxDB team is working on a Parquet‑based migration solution for InfluxDB V3; consider waiting if you are dealing with petabytes of data.
```

Comment:
Even terabytes may warrant waiting for a parquet-based migration. Not many LiveAnalytics customers have petabytes currently, but there are some.